apps/guide/src/content/04-popular-topics/05-collectors.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Collectors
+category: Popular topics
+---
+
+# Collectors
+
+## Message collectors
+
+<p><DocsLink type="class" parent="Collector"><code>Collector</code>s</DocsLink> are useful to enable your bot to obtain *additional* input after the first command was sent. An example would be initiating a quiz, where the bot will "await" a correct response from somebody.</p>
```

Comment:
`<DocsLink type="class" parent="Collector"><code>Collector</code>s</DocsLink>` doesn't seem to work, The `<code>Collector</code>s` is ignored by the component and just shows an external link titled "Collector"